### ISSUE TITLE
Create some pre-built configs

### ISSUE THREAD

**Issue by NuroDev** [sqshq/sampler#24]
Currently there is a single example config file which serves as a great starting point, but it would be nice to have a few pre-built configs for people to drop in and go.

Some examples:
 - One designed entirely around Docker and monitoring container metrics.
 - Another designed to look pretty and having an overview of system metrics.

**Comment by sqshq** [comment]
I agree, that is a good idea. Actually my plan was to have a Dockerfile per each most popular linux distribution, with all infrastructure installed for [Real-world recepies](https://github.com/sqshq/sampler#real-world-recipes) section. And as you suggested, we might have a number of config files there to use - for test and documentation at the same time.

That way after each commit it would be easy to test that nothing everything works as expected.
